great_expectations/dataset/pandas_dataset.py

```diff
@@ -118,7 +118,7 @@ def inner_wrapper(  # noqa: PLR0913, PLR0912
                 nonnull_values[boolean_mapped_success_values == False].index
             )
 
-            if "output_strftime_format" in kwargs:
+            if kwargs.get("output_strftime_format") is not None:
```

Comment:
We now get the keys for null values, e.g. kwargs can look like {"output_strftime_format": None}`